pages/common/git-fame.md

```diff
@@ -0,0 +1,28 @@
+# git fame
+
+> Pretty-print Git repository contributions.
+> More information: <https://github.com/casperdcl/git-fame>.
+
+- Calculate contributions for the current Git repository:
+
+`git fame`
+
+- Exclude files/directories that match the specified regular expression:
+
+`git fame --excl "{{regular_expression}}"`
+
+- Calculate contributions made after the specified date:
+
+`git fame --since "{{3 weeks ago|2021-05-01}}"`
```

Comment:
Instead of giving options, should we give a concrete run-able example ?
Thoughts ?

```suggestion
`git fame --since "{{2021-05-01}}"`

- Calculate contributions made relative to 'today':

`git fame --since "{{3 weeks ago}}"`
```

Reply:
I think having two almost exact exmaples could make it more difficult. Both `3 weeks ago` and `2021-05-01` are self-explaining.

Reply:
I agree that this looks a bit weird, but I though as @marchersimon did: they both _are_ completely self explanatory.  Also, this command has more functionality that I didn't document, and adding a new example to demonstrate essentially the same behavior as an	existing example seems like a waste.